src/rebar_state.erl

```diff
@@ -438,13 +438,13 @@ namespace(#state_t{namespace=Namespace}) ->
 namespace(State=#state_t{}, Namespace) ->
     State#state_t{namespace=Namespace}.
 
-packages(State=#state_t{packages=undefined}) ->
-    rebar_packages:get_packages(State);
-packages(#state_t{packages=Packages}) ->
-    Packages.
+packages_graph(State=#state_t{packages_graph=undefined}) ->
+    rebar_packages:get_packages_graph(State);
```

Comment:
Thanks, I've rebased against your changes and updated `packages_graph` to work the same as you did for `packages`.